README.md

```diff
@@ -157,12 +157,19 @@ Buildkite Agent for versions of macOS prior to those listed above.
 
 ## Contributing
 
-1. Fork it
-1. Create your feature branch (`git checkout -b my-new-feature`)
+1. Fork this repo
+1. Create a feature branch with a nice name (`git checkout -b my-new-feature`)
+1. Write your code!
+    - Please make sure your code is correctly formatted by running `go fmt ./...`, and that the tests are passing by running `go test ./...`
 1. Commit your changes (`git commit -am 'Add some feature'`)
-1. Push to the branch (`git push origin my-new-feature`)
+    - In an ideal world we have [atomic commits](https://www.pauline-vos.nl/atomic-commits/) and use [Tim Pope-style commit messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), but so long as it's clear what's happening in your PR, that's fine. We try to not be super persnickety about these things.
+1. Push to your the branch (`git push origin my-new-feature`)
 1. Create new Pull Request
 
+The agent wranglers at Buildkite will review your PR, and start a CI build for it. For security reasons, we don't automatically run CI against forked repos, and a human will review your PR prior to its CI running.
```

Comment:
> agent wranglers

Nice! I look forward to us putting this on our business cards 🪪

Reply:
i'm looking forward to getting the business cards in the first place :P